src/com/dynatrace/loadrunner/CommandLineParser.java

```diff
@@ -0,0 +1,169 @@
+package com.dynatrace.loadrunner;
+
+import java.util.HashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CommandLineParser {
+
+	public HashMap<String, String> arguments;
+
+	public CommandLineParser(String[] args) {
+
+		arguments = new HashMap<String, String>();
+
+		String key = null;
+		String value = null;
+		int index = 0;
+
+		for (int i = 0; i < args.length; i++) {
+
+			if (args[i].startsWith("--")) {
+				key = args[i].substring(2);
+			} else if (args[i].startsWith("-")) {
+				key = args[i].substring(1);
+			} else {
+				arguments.put(args[i], null);
+				continue;
+			}
+			index = key.indexOf('=');
+
+			if (index == -1) {
+				if ((i + 1) < args.length) {
+					if (args[i + 1].charAt(0) != '-') {
+						arguments.put(key, args[i + 1]);
+						i++;
+					} else {
+						arguments.put(args[i], null);
+					}
+				} else {
+					arguments.put(args[i], null);
+				}
+			} else {
+				value = key.substring(index + 1);
+				key = key.substring(0, index);
+				arguments.put(key, value);
+			}
+		}
+	}
+
+	public String getValue(String key) {
+		if (InputUtils.isValid(key)) {
+			return arguments.get(key);
+		}
+		return null;
+	}
+
+	public boolean containsKey(String key) {
+
+		if (InputUtils.isValid(key)) {
+
+			if (arguments.get(key) != null) {
+				return true;
+			} else {
+				return false;
+			}
+		}
+		return false;
+
+	}
+
+}
+
+class InputUtils {
```

Comment:
class has only static methods - use private constructor to prevent senseless object creation